inputJsonFiles/ConnectionMap/1PC1worker.json

```diff
@@ -0,0 +1,34 @@
+{
+    "connectionsMap":
+    {
+        "c1": {
+        "mainServer": "r1",
+        "f1": "r1"
+        },
+        "f1": {
+        "mainServer": "r1",
+        "c1": "r1"
+        },
+
+        "s1": {
+        "mainServer": "r1",
+        "c1": "r1"
+        },
+        "mainServer": {
```

Comment:
Can we eliminate the need for duplicate configuration? 
Definitions here define communication in both directions 
Or we can just use adjacency list of tuples
{c1,r1}
{c1,r2}
...etc 

Moreorver, mainServer is connected to all components in the system as we present in Nerlnet architecture since mainServer is actually the monitor of all instances in Nerlnet network. 
The server API as well is always connected to the mainServer, we just define the serverAPI the IP of the main server in configuration. I want the connections map to illustrate the distributed machine learning experiment, main server and serverAPI are not part of the experiment but just supervisors. 

@kapelnik